ocs_ci/deployment/baremetal.py

```diff
@@ -190,6 +191,21 @@ def deploy_prereq(self):
                 assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"
             else:
                 raise RhcosImageNotFound
+
+            if ocp_version == "4.6":
+                # Download metal_bios
+                rootfs_image_path = constants.coreos_url_prefix + image_data['live_rootfs_url']
```

Comment:
use os.path.join()

Reply:
Done

Reply:
os.path.join doesn't work here
```import os
a="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos"
b="/pre-release/latest-4.6/rhcos-live-rootfs.x86_64.img"
print(os.path.join(a,b))                                                                                                                                                                                                              
/pre-release/latest-4.6/rhcos-live-rootfs.x86_64.img
```

Reply:
you need to remove slash in the beginning ( b="pre-release/latest-4.6/rhcos-live-rootfs.x86_64.img" )